akka-http-testkit/src/main/scala/akka/http/javadsl/testkit/RouteTest.scala

```diff
@@ -33,7 +34,7 @@ abstract class RouteTest extends AllDirectives with WSTestRequestBuilding {
   implicit def materializer: Materializer
   implicit def executionContext: ExecutionContextExecutor = system.dispatcher
 
-  protected def awaitDuration: FiniteDuration = 3.seconds
+  protected def awaitDuration: FiniteDuration = new JavaTestKit(system).dilated(1.seconds).asInstanceOf[FiniteDuration];
```

Comment:
I would leave the default timeout to 3 seconds.

I'm not so sure about using this class, as it is deprecated